wp-cli/civicrm.php

```diff
@@ -1163,7 +1163,7 @@ private static function parseDSN( $dsn ) {
 
 			// Find protocol and hostspec
 
-			if ( preg_match( '|^( [^( ]+ )\( ( .*? )\ )/?( .*? )$|', $dsn, $match ) ) {
+			if ( preg_match( '|^([^(]+)\((.*? )\)/?(.*?)$|', $dsn, $match ) ) {
```

Comment:
Is there one extra space to remove (after the first question mark)?

Reply:
missed that.  Thanks.  Updating the PR